src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -310,6 +322,9 @@ export default {
             this.removeSelectable = this.openmct.selection.selectable(
                 this.$el, this.context, this.immediatelySelect || this.initSelect);
             delete this.immediatelySelect;
+            this.unsubscribeFromStaleness = this.openmct.telemetry.subscribeToStaleness(this.domainObject, (isStale) => {
```

Comment:
Oh man.. I agree. I was holding off just because I thought it might be a pain, but it could be a REALLY useful mixin for anything using staleness. I guess what I'm trying to say is... damn it.

Reply:
alright! that was not too bad at all. done!